src/main/java/io/quarkus/search/app/entity/Guide.java

```diff
@@ -45,27 +50,54 @@ public class Guide {
     @KeywordField
     public String origin;
 
-    @FullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS)
-    @FullTextField(name = "title_autocomplete", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT)
-    @KeywordField(name = "title_sort", normalizer = AnalysisConfigurer.SORT, searchable = Searchable.NO, sortable = Sortable.YES)
-    @Column(length = Length.LONG)
-    public String title;
-
-    @FullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS)
-    @FullTextField(name = "summary_autocomplete", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT)
-    @Column(length = Length.LONG32)
-    public String summary;
+    @I18nFullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS, localization = {
+            @Localization(language = "en"),
+            @Localization(language = "es"),
+            @Localization(language = "ja")
+    })
+    @I18nFullTextField(name = "title_autocomplete", localization = {
+            @Localization(language = "en", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT),
+            @Localization(language = "es", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT),
+            @Localization(language = "ja", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT)
```

Comment:
This makes me think a lot of this will feel redundant, when you actually use one analyzer per language...

I guess that, alternatively, you could just expect analyzer name prefixes to be passed to `I18nFullTextField`, and have the bridge append a language code.

E.g.:

```
@I18nFullTextField(name = "title_autocomplete", analyzer = AnalysisConfigurer.AUTOCOMPLETE_PREFIX, searchAnalyzer = AnalysisConfigurer.DEFAULT_PREFIX)
```

Would use `AnalysisConfigurer.AUTOCOMPLETE_PREFIX + "_en"` for English, `AnalysisConfigurer.AUTOCOMPLETE_PREFIX + "_es"` for Spanish, etc.